google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -284,7 +284,7 @@ long getRandomizePositionQPSThreshold() {
     return randomizePositionQPSThreshold;
   }
 
-  boolean getUseMultiplexedSession() {
+  public boolean getUseMultiplexedSession() {
```

Comment:
This is made public since its needed for connection API tests.

Reply:
nit: can we mark it with `@InternalApi` until the setter has also been made public? Or maybe `@VisibleForTesting`?

Reply:
Done